e2e/test/TwinE2ETests.cs

```diff
@@ -103,17 +103,11 @@ public async Task Twin_ServiceSetsDesiredPropertyAndDeviceReceivesItOnNextGet_Mq
             await Twin_ServiceSetsDesiredPropertyAndDeviceReceivesItOnNextGet(Client.TransportType.Mqtt_Tcp_Only).ConfigureAwait(false);
         }
 
+        [Ignore] // TODO: #682 - intermittently failing with parsing error.
         [TestMethod]
```

Comment:
Why is the test modification needed? And why are you adding ignore?

Reply:
oh my... it got in from some other check-ins.. I'll remove it